teamBangazon/BangazonCLI
# Payment Type tests

### Changes Made

The following files were added/modified:

Bangazon.Tests/AddPaymentTypeStringBuilderShould.cs
Bangazon/Managers/AddPaymentTypeManager.cs
Bangazon/Models/PaymentType.cs
Bangazon/StringBuilders/AddPaymentTypeStringBuilder.cs

AddPaymentTypeStringBuilderShould.cs tests run and pass.

|||||||||||

Created: ActiveCustomerMenu.cs
ActiveCustomerMenuShould.cs
Modified: AddCustomerManagerShould.cs(added db parameter to test - line: 18)
DatabaseInterface.cs(changed spelling error - line: 102)
MainMenu.cs(Added ActiveCustomer Select to MainMenu)

### Steps to Test

```
git checkout master
git fetch --all
git checkout kw-paymenttype2
```

Ensure tests run properly.
Create customer select active customer and then add a payment type.  Please verify that all are created uniquely and the payment type's customerId is correctly linking to the customer table.
### Ref URL
#3 
https://github.com/teamBangazon/BangazonCLI/issues/2
